### ISSUE TITLE
[DATASET] CULane

### ISSUE THREAD

**Issue by alessandrodicosola** [alessandrodicosola/FellowshipResearch#7]
## [CULane](https://xingangpan.github.io/projects/CULane.html)

Description: CULane is a large scale challenging dataset for academic research on traffic lane detection with 88880 images for training set, 9675 for validation set, and 34680 for test set. The test set is divided into normal and 8 challenging categories, which correspond to the 9 examples above.

Annotations: 2d points, semantic segmentation
